### PR TITLE
Anna and Tobi training to query the spoonacular API

### PR DESCRIPTION
**Pull request by tobiasbach**
We defined a URL, built a hash, converted a hash to JSON. Finally we queried the API with a POST request, had a look at our response data and used it to generate some output.

### CODE REVIEW

File: cli.rb [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+        "0.5 cup vinegar"
+    ],
+    "instructions": "Get a bowl, put all the ingredients in and arrange it nicely."
+}
+
+# Querying the API with our URL and recipe hash - converted to JSON
+content = Net::HTTP.post(URI(uri), recipe.to_json)
+
+# Parsing the response body (JSON file) and printing some output to the terminal
+if JSON.parse(content.body)["vegetarian"] == true
```
Comment (varamezova) [inline cli.rb:29]: Emojis weg!

File: cli.rb [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+    puts "🥗 Guten Appetit, das darfst du essen! 🥣"
+else
+    puts "🐥 Finger weg! Töte keine Tiere! 🐷"
```
Comment (varamezova) [inline cli.rb:32]: Emojis weg!!!

File: cli.rb [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
-puts(content)
+# Defining the URL we want to query
+# TODO: Remove API key from the string. Import it from the config file and use a variable instead (string interpolation).
+uri = "https://api.spoonacular.com/recipes/analyze?apikey=5e0356f3b7bd454886811a57765cecf9"
+
+# Building a recipe hash
+# TODO: Reduce the recipe to a minimum. Which parts of it do we really need?
+# TODO: May we want to enable the user to enter his own recipe?
+recipe = {
+    "title": "Salty Salad",
```
Comment (varamezova) [inline cli.rb:13]:
```suggestion
    title: "Salty Salad",
```

File: cli.rb (hunk 2 above)
Comment (Azure7111) [reply cli.rb:32]: The emojis work, in a docker container at least.

So I think we can keep them.

Do you have a preference either way?

File: cli.rb [hunk 4]
```diff
@@ -4,7 +4,8 @@
 
 # Defining the URL we want to query
 # TODO: Remove API key from the string. Import it from the config file and use a variable instead (string interpolation).
-uri = "https://api.spoonacular.com/recipes/analyze?apiKey=5e0356f3b7bd454886811a57765cecf9"
+key = File.read("config").split[1]
+uri = "https://api.spoonacular.com/recipes/analyze?apiKey="+key
```
[the remark this answers is not in the document]
Comment (tobiasbach) [reply cli.rb:8]: + If we have a working solution the TODO comment can be removed as well.

**Review by Azure7111 — APPROVED** [review]

**Pull request merged**